x/gov/keeper/proposal.go

```diff
@@ -54,7 +55,10 @@ func (keeper Keeper) SubmitProposal(ctx sdk.Context, messages []sdk.Msg, metadat
 		if msg, ok := msg.(*v1.MsgExecLegacyContent); ok {
 			cacheCtx, _ := ctx.CacheContext()
 			if _, err := handler(cacheCtx, msg); err != nil {
-				return v1.Proposal{}, sdkerrors.Wrap(types.ErrNoProposalHandlerExists, err.Error())
+				if errors.Is(types.ErrNoProposalHandlerExists, err) {
```

Comment:
Reverts the reversion of https://github.com/cosmos/cosmos-sdk/pull/16331 (cc @yihuang), as everyone needs to upgrade tomorrow to v0.46.13 when we've released the fix.

Reply:
Because the security fix is a breaking one?

Reply:
I have the same question

Reply:
the security fix seems not a breaking one, so we hope the sdk `0.46.13` is not breaking as well, against `< 0.46.12`, since there seems no good way to do an emergent breaking upgrade yet.

Reply:
Oh dearsies okay I'm reaching out to different teams right now to figure out if the chains need to go down. Thank you so much for your feedback @yihuang

Reply:
please dont cause chaos @faddat teams should reach out the people that understand the bug. 

lets revert the change to the non breaking version for now @julienrbrt thats my bad

Reply:
Alright, let me make a list of people in v0.46.12 then 👍🏾